src/odemis/driver/actuator.py

```diff
@@ -666,6 +667,41 @@ def _get_rot_matrix(self, invert=False):
             rotation *= -1
         return RigidTransform(rotation=rotation).matrix
 
+    def _getAxesRange(self):
+        """
+        Calculate the axes range for position VA based on the conversion matrix
+        from dep's positon.
```

Comment:
If I read this docstring it is not entiry clear how to interpret it, what is dep's position for example? And the conversion matrix?
There is also a typo in there: positon -> position

Reply:
I have updated the doc string.